FUNDING.rst

```diff
@@ -0,0 +1,20 @@
+Funding
```

Comment:
I love this. We should also include OpenEarthscape and CSDMS 4 awards.

Reply:
I've added both to `FUNDING.rst`.